modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -1757,6 +1757,15 @@ void NativeReloadNode::_notification(int p_what) {
 				}
 
 				for (Map<String, Set<NativeScript *>>::Element *U = NSL->library_script_users.front(); U; U = U->next()) {
+					// backported from godotengine/godot#46514
```

Comment:
That's relevant info for the PR description but not so much for the merged code.

On the other hand, you can give proper credits by adding this to the commit message:
```
Co-authored-by: geekrelief <geekrelief@gmail.com>
```